Gifski/util.swift

```diff
@@ -938,6 +938,22 @@ extension NSView {
 		addSubview(view)
 		view.center(inView: superview!)
 	}
+
+	func pinToSuperview(insets: NSEdgeInsets = .zero) {
```

Comment:
This is not used anywhere.

Reply:
Will be! Replacing the manual frame estimation with auto-layout as we speak.